test/node/conversation-items.js

```diff
@@ -18,18 +18,15 @@ describe('Conversation Items', () => {
         user = await client.logon();
         client2 = new Circuit.Client(config.bot2);
         user2 = await client2.logon();
+        const topic = `${Date.now()}a`;
+        conversation = await client.createGroupConversation([user2.userId], topic);
```

Comment:
Could you use the preparation conversation here?

Reply:
done